### PR TITLE
Add more French translations

### PR DESCRIPTION
**Pull request by AnnikaCodes**
All credit for the translations goes to Adri - I'm just adding this to the repo.

### CODE REVIEW

File: translations/french.json [hunk 1]
```diff
@@ -129,46 +129,46 @@
...
-		"You are not allowed to use filtered words in polls.": "",
-		"Not enough arguments for /poll new.": "",
-		"Too many options for poll (maximum is 8).": "",
-		"There are duplicate options in the poll.": "",
-		"${user.name} queued a poll.": "",
-		"A poll was started by ${user.name}.": "",
-		"The queue is already empty.": "",
-		"Cleared poll queue.": "",
-		"Room \"${roomid}\" not found.": "",
+		"The announcement has ended.": "L'annonce est terminée",
```
Comment (Alumn2000) [inline translations/french.json:132]:
```suggestion
		"The announcement has ended.": "L'annonce est terminée.",
```

File: translations/french.json [hunk 2]
```diff
@@ -129,46 +129,46 @@
...
+		"Battles do not support announcements.": "Les combats ne permettent pas de faire des annonces.",
+		"You are not allowed to use filtered words in announcements.": "Tu n'es pas autorisé à utiliser des mots filtrés dans les annonces.",
+		"There is already a poll or announcement in progress in this room.": "Il y a déjà un sondage ou une annonce dans cette room.",
+		"An announcement was started by ${user.name}.": "Une annonce a été lancée par ${pseudo}.",
+		"There is no announcement running in this room.": "Il n'y a actuellement pas d'annonce dans cette room.",
+		"There is no timer to clear.": "Il n'y a pas de minuteur à supprimer. ",
```
Comment (Alumn2000) [inline translations/french.json:138]:
```suggestion
		"There is no timer to clear.": "Il n'y a pas de minuteur à supprimer.",
```

File: translations/french.json [hunk 3]
```diff
@@ -129,46 +129,46 @@
...
+		"The announcement timer was turned off.": "Le minuteur de l'annonce a été désactivé.",
+		"Invalid time given.": "Temps donné invalide.",
+		"The announcement timer is off.": "Le minuteur de l'annonce n'est pas actif.",
+		"The announcement was ended by ${user.name}.": "L'annonce a été terminée par ${user.name}.",
+		"Accepts the following commands:": "Accepte les commandes suivantes:",
```
Comment (Alumn2000) [inline translations/french.json:143]:
```suggestion
		"Accepts the following commands:": "Accepte les commandes suivantes :",
```

File: translations/french.json [hunk 4]
```diff
@@ -129,46 +129,46 @@
...
+		"You are not allowed to use filtered words in polls.": "Tu n'es pas autorisé à utiliser des mots filtrés dans les annonces.",
+		"Not enough arguments for /poll new.": "Pas assez d'arguments pour /poll new.",
+		"Too many options for poll (maximum is 8).": "Il y a trop d'options pour un sondage (le maximum est de 8).",
+		"There are duplicate options in the poll.": "Il y a plusieurs options identiques dans le sondage.",
+		"${user.name} queued a poll.": "${pseudo} a mis un sondage en attente.",
+		"A poll was started by ${user.name}.": "Un sondage a été lancé par ${pseudo}.",
+		"The queue is already empty.": "La file d'attente est déjà vide.",
+		"Cleared poll queue.": "La file d'attente des sondages a été supprimée.",
+		"Room \"${roomid}\" not found.": "La salle \"${roomid}\ n'a pas été trouvée.",
 		"Can't delete poll at slot ${slotString} - \"${slotString}\" is not a number.": "",
```
Comment (Alumn2000) [inline translations/french.json:156]:
```suggestion
		"Can't delete poll at slot ${slotString} - \"${slotString}\" is not a number.": "Impossible de supprimer le sondage en position ${slotString} - \"${slotString}\" n'est pas un nombre.",
```

File: translations/french.json [hunk 5]
```diff
@@ -129,46 +129,46 @@
...
-		"The poll timer was set to ${timeout} minute(s) by ${user.name}.": "",
-		"The poll timer is on and will end in ${poll.timeoutMins} minute(s).": "",
-		"The poll timer is off.": "",
-		"The poll was ended by ${user.name}.": "",
-		"Queued polls:": "",
-		"Refresh": "",
-		"No polls queued.": "",
-		"#${number} in queue": ""
+		"There is no poll in queue at slot ${slot}.": "Il n'y a pas de sondage dans la file d'attente en position ${slot}.",
+		"(${user.name} deleted the queued poll in slot ${slot}.)": "${pseudo} a supprimé le poll en attente en position ${slot}.",
```
Comment (Alumn2000) [inline translations/french.json]:
```suggestion
		"(${user.name} deleted the queued poll in slot ${slot}.)": "(${pseudo} a supprimé le sondage en attente en position ${slot}.)",
```

File: translations/french.json [hunk 6]
```diff
@@ -129,46 +129,46 @@
...
+		"There is no poll running in this room.": "Il n'y a pas de sondage actuellement dans cette room.",
+		"To vote, specify the number of the option.": "Pour voter, spécifiez le numéro de l'option.",
+		"Option not in poll.": "L'option n'est pas dans le sondage.",
+		"The poll timer was turned off.": "Le minuteur du sondage a été désactivé.",
+		"The queued poll was started.": "Le sondage en attente a été lancé.",
+		"The poll timer was turned on: the poll will end in ${timeout} minute(s).": "Le minuteur du sondage a été activé: le sondage va se terminer dans ${timeout} minute(s).",
```
Comment (Alumn2000) [inline translations/french.json:164]:
```suggestion
		"The poll timer was turned on: the poll will end in ${timeout} minute(s).": "Le minuteur du sondage a été activé : le sondage va se terminer dans ${timeout} minute(s).",
```

File: translations/french.json [hunk 7]
```diff
@@ -129,46 +129,46 @@
...
+		"The poll timer was set to ${timeout} minute(s) by ${user.name}.": "Le minuteur du sondage a été fixé à ${timeout} minute(s) par ${user.name}.",
+		"The poll timer is on and will end in ${poll.timeoutMins} minute(s).": "Le minuteur du sondage est activé et se terminera dans ${poll.timeoutMins} minute(s).",
+		"The poll timer is off.": "Le minuteur du sondage est désactivé.",
+		"The poll was ended by ${user.name}.": "Le sondage a été terminé par ${user.name}.",
+		"Queued polls:": "Sondages en attente.",
```
Comment (Alumn2000) [inline translations/french.json:169]:
```suggestion
		"Queued polls:": "Sondages en attente :",
```

File: translations/french.json [hunk 8]
```diff
@@ -129,46 +129,46 @@
...
+		"Refresh": "Rafraîchir",
+		"No polls queued.": "Pas de sondage en attente.",
```
Comment (Alumn2000) [inline translations/french.json:171]:
```suggestion
		"No polls queued.": "Pas de sondages en attente.",
```

File: translations/french.json (hunk 3 above)
Comment (AnnikaCodes) [reply translations/french.json:143]: Is this intentional?

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**